patches/ungoogled-chromium/windows/windows-fix-clang-format-exe.patch

```diff
@@ -11,13 +11,14 @@
  
  def parse_options():
      parser = optparse.OptionParser(usage="%prog [options] TASK...")
-@@ -53,6 +56,10 @@ def parse_options():
+@@ -53,6 +57,11 @@ def parse_options():
  
  
  def main():
 +    # This fix build issue on Windows caused by missing google dependencies.
 +    if os.name == 'nt':
-+        shutil.copy('../../third_party/llvm-build/Release+Asserts/bin/clang-format.exe', '../../buildtools/win')
++        if not os.path.exists('../../buildtools/win/clang-format.exe')
```

Comment:
Is this even valid syntax without the colon `:` at the end?

Reply:
Oh right

Reply:
This is fixed now